web/src/pages/Settings/Streams/StreamModal/StreamModal.jsx

```diff
@@ -133,19 +133,6 @@ InternetRadioSearch.propTypes = {
     onChange: PropTypes.func.isRequired,
 };
 
-function validateInput(streamFields, streamTemplate) {
-    streamTemplate['fields'].filter((f) => f.required).map( (f) => {
```

Comment:
Does this move all required field validation to the api?

Reply:
yes